src/hotspot/share/oops/klass.cpp

```diff
@@ -260,7 +282,8 @@ Klass::Klass() : _kind(UnknownKlassKind) {
 // The constructor is also used from CppVtableCloner,
 // which doesn't zero out the memory before calling the constructor.
 Klass::Klass(KlassKind kind) : _kind(kind),
-                           _shared_class_path_index(-1) {
+                               _secondary_supers_bitmap(SECONDARY_SUPERS_BITMAP_EMPTY),
```

Comment:
Looks like it is redundant since metaspace allocation already initializes memory with zeroes .